brave/brave-core
# Fixes linter for native ledger

Partial https://github.com/brave/brave-browser/issues/3413

- [ ] switch if statements
- [ ] std:bind alignment

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (`npm test brave_unit_tests && npm test brave_browser_tests`) on

## Test Plan:
- make sure that `cpplint --recursive vendor/bat-native-ledger/*` returns only ~220 errors (before it was 800+)
- make sure that code builds

## Reviewer Checklist:
